app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,78 @@
+require './config/environment'
+
+class TweetsController < ApplicationController
+
+#All Tweets
+  get '/tweets' do
+    if logged_in?
+      @user = current_user
+      erb :'/tweets/tweets'
+    else
+      redirect to "/login"
+    end
+  end
+
+#New Tweet
+  get '/tweets/new' do
+    if logged_in?
+      erb :'/tweets/create_tweet'
+    else
+      redirect to "/login"
+    end
+  end
+
+#New Tweet- Form Submit
+  post '/tweets' do
+    if params[:content] == ""
+      redirect '/tweets/new'
+    else
+      @tweet = Tweet.create(:content => params["content"])
+      @tweet.user_id = current_user.id
+      @tweet.save
+      redirect to "/tweets/#{@tweet.id}"
+    end
+  end
+
+#Show Tweet
+  get '/tweets/:id' do
+    if logged_in?
+      @tweet = Tweet.find(params[:id])
+      erb :'/tweets/show_tweet'
+    else
+      redirect to "/login"
+    end
+  end
+
+#Edit Tweet
+  get '/tweets/:id/edit' do
+    if logged_in?
+      @tweet = Tweet.find(params[:id])
+      erb :'/tweets/edit_tweet'
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?